apps/developer-hub/src/components/IntegrationCard/index.tsx

```diff
@@ -22,35 +26,59 @@ const colorClasses = {
     text: "text-purple-600 dark:text-purple-400",
     hoverText: "group-hover:text-purple-600 dark:group-hover:text-purple-400",
   },
+  yellow: {
+    bg: "bg-yellow-100 dark:bg-yellow-900",
+    text: "text-yellow-600 dark:text-yellow-400",
+    hoverText: "group-hover:text-yellow-600 dark:group-hover:text-yellow-400",
+  },
 };
 
-export const IntegrationCard = ({
+export function IntegrationCard({
   href,
   icon,
   title,
   description,
   colorScheme = "blue",
-}: IntegrationCardProps) => {
+  external,
+  showArrow = true,
+}: IntegrationCardProps) {
   const colors = colorClasses[colorScheme];
+  const Wrapper = external ? "a" : Link;
```

Comment:
I strongly recommend against doing this kind of pattern.  It's generally unneeded for avoiding repetition, and doing things like this causes messy typing, and it generally makes the code harder to predict & understand.  I used to use this pattern and have nearly always moved away from it.

Instead, what I recommend doing is something along these lines:

```tsx
const MyComponent = ({ external, ...props }) =>
  external
    ? <a {...commonProps(props)} />
    : <Link {...commonProps(props)} />

const commonProps = (props) => ({
  href: props.href,
  className: "not-prose ...",
  children: (
    <div>
      ...
    </div>
  )
});